### PR TITLE
DPC-732: Basic performance test suite for /Token endpoints

### PR DESCRIPTION
**Pull request by embh**
### Fixes [DPC-732](https://jira.cms.gov/browse/DPC-732)

DPC is setting up a basic performance test suite for attribution endpoints.

### Proposed Changes

This pull request adds performance tests for the /Token endpoints.

### Change Details

Tests include:
* POST /Token
* POST /Token/validate
* POST /Token/auth
* GET /Token
* GET /Token/{id}
* DELETE /Token/{id}

The new tests are in token.go, and the existing /Key endpoint tests have been moved into key.go.

### Security Implications

This contains tests only. No security impacts.

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

Screen Shot 2020-09-24 at 5 18 38 PM

### Feedback Requested

Any

### CODE REVIEW

File: dpc-testing/performance/key.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+func newPOSTKeyTargeter(accessToken string) vegeta.Targeter {
+	return func(t *vegeta.Target) error {
+		t.Method = "POST"
+		t.URL = fmt.Sprintf("%s/Key", apiURL)
+		t.Header = map[string][]string{
+			"Content-Type":  {"application/json"},
+			"Authorization": {fmt.Sprintf("Bearer %s", accessToken)},
+		}
+
+		pubKeyStr, _, signature := generateKeyPairAndSignature()
```
Comment (jonfulk) [inline dpc-testing/performance/key.go:57]: This is probably due to my limited knowledge of Go, but I'm not finding the definition of `generateKeyPairAndSignature`

**Review by jonfulk — COMMENTED** [review]
I don't feel qualified to approve this one, but I looked over it and didn't notice any issues with the approach.

File: dpc-testing/performance/key.go (hunk 1 above)
Comment (embh) [reply dpc-testing/performance/key.go:57]: Thanks for taking a look! That function is in setup.go, in the same `package main`: https://github.com/CMSgov/dpc-app/blob/master/dpc-testing/performance/setup.go#L78

**Review by dhgreene — APPROVED** [review]
Works great locally and in `dev`!

**Review by SalGutierrez — APPROVED** [review]
LGTM, would be nice to have a challenge exercise for performance testing. 👍

**Pull request merged**